contracts/cw721-base/src/error.rs

```diff
@@ -1,13 +1,14 @@
 use cosmwasm_std::StdError;
+use cw_ownable::OwnershipError;
 use thiserror::Error;
 
 #[derive(Error, Debug, PartialEq)]
 pub enum ContractError {
-    #[error("{0}")]
+    #[error(transparent)]
```

Comment:
Is there any difference between `"{0}"` and `transparent`?

Reply:
this is the way that the `thiserror` docs suggest passing through errors, though the actual serialization doesn't change. i feel this is a little more explicit and that's what i do in other contracts, though i don't have an opinion on what we use in this repo if we'd like to stick to `{0}`.